azure/v15.0.0/release.diff

```diff
@@ -0,0 +1,58 @@
+    version: 1.3.1                                                     version: 1.3.1
+  - name: metrics-server                                             - name: metrics-server
+    version: 1.3.0                                                     version: 1.3.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.10.1                                                    version: 1.10.1
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.7.2                                                     version: 1.7.2
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.20.2                                                    version: 1.20.2
+  - name: azure-scheduled-events                                     - name: azure-scheduled-events
+    version: 0.4.0                                                     version: 0.4.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 4.4.0                                                     version: 4.4.0
+  - name: azure-operator                                             - name: azure-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 5.7.0                                                     version: 5.7.0
+  - name: cert-operator                                              - name: cert-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 1.0.1                                                     version: 1.0.1
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.27.1                                                    version: 0.27.1
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.20.6                                                    version: 1.20.6
+  - name: containerlinux                                             - name: containerlinux
+    version: 2605.12.0                                                 version: 2605.12.0
+  - name: calico                                                     - name: calico
+    version: 3.15.3                                                    version: 3.15.3
+  - name: etcd                                                       - name: etcd
+    version: 3.4.14                                                    version: 3.4.14
+  date: "2021-06-14T11:40:38Z"                                  |    date: "2021-05-24T11:40:38Z"
```

Comment:
I think this is wrong. The left side is the previous release, the right side is the new one. You should have bumped the right side date

Reply:
![](https://ollesbloggtechome.files.wordpress.com/2019/04/test.gif)